logger_cb.go

```diff
@@ -0,0 +1,98 @@
+package libbpfgo
+
+/*
+#include <stdlib.h>
+#include <bpf/libbpf.h>
+*/
+import "C"
+
+import (
+	"regexp"
+	"strings"
+
+	"github.com/aquasecurity/tracee/pkg/logger"
```

Comment:
How about define an `interface` instead of depend on `tracee`?

Reply:
I think it will be hard to use an interface here because we do need the logger levels.
We can use the `zap` levels, but it will make an ununified interface.
Maybe we should create a standalone package under Tracee (like `types`) for it, or event an new project.

Reply:
> Maybe we should create a standalone package under Tracee (like types) for it, or create a new project.

This was my initial intent way back. If this is ok, we can merge it in next, start to test it and do issue logger, as an own project, to a farther milestone. WDYT?

Reply:
I agree.

Reply:
- https://github.com/aquasecurity/tracee/issues/2599

Reply:
As https://github.com/aquasecurity/tracee/pull/2600#pullrequestreview-1253549004 concluded, we'll tackle this creating a libbpfgo.Logger interface.

Reply:
> As [aquasecurity/tracee#2600 (review)](https://github.com/aquasecurity/tracee/pull/2600#pullrequestreview-1253549004) concluded, we'll tackle this creating a libbpfgo.Logger interface.

@rafaeldtinoco thanks for reviewing this.

Returing to this soon.

Reply:
Done. Putting this as ready for review.